### DISCUSSION TITLE
Issue of installation

### DISCUSSION THREAD

**Discussion by GuChad369** [pgmoneta/pgmoneta#213] [Q&A]
When I attempted to install `pgmontea`, I encountered two problems:
1. When I added master-key, I had to input a string longer than 9 characters to proceed. Otherwise, it kept asking for the master key. Is it a default requirement for the password length to be greater than 9 characters?
![312874118-6f17b7db-e1b5-4e9f-9c59-f9e9c4c03100](https://github.com/pgmoneta/pgmoneta/assets/113565433/0300c6f8-bb65-471f-afe3-18cf55347016)

2. During the backup, there was no influence on the details; the used space always showed as 0.
update database:
![image](https://github.com/pgmoneta/pgmoneta/assets/113565433/1d2b4370-e794-44ce-ad6d-014ce5841ff2)
check backup:
![image](https://github.com/pgmoneta/pgmoneta/assets/113565433/5d160fb5-7da0-4917-93c5-dbcbd4a67072)
What are the potential problems here?

**Comment by Jubilee101** [comment]
Hi, it doesn't look like you've made a backup. Run `pgmoneta-cli -c pgmoneta.conf backup primary`.

You should be able to see something like
![image](https://github.com/pgmoneta/pgmoneta/assets/89931362/b0d766b9-7c86-4a31-8ff3-39d07db5246f)

**Comment by GuChad369** [comment]
Yes, I ran it, but it didn't work. I'm trying to figure out what's wrong here.

**Comment by Jubilee101** [comment]
You can check the logs on pgmoneta side. You can also check the logs in postgres server.

**Comment by GuChad369** [comment]
If you encounter the master-key issue like I did, ensure that the master key is at least 8 characters long. For more information, check `pgmoneta/doc/GETTING_STARTED.md`

**Comment by GuChad369** [comment]
The second issue has also been fixed. If you encounter this, it may be related to authentication. You can update your replication password. For details, check your logs to identify the problem.
